doc/installingLocal.md

```diff
@@ -39,7 +47,22 @@ docker run -it --rm -p 3000:3000 -v /path/to/PrairieLearn:/PrairieLearn prairiel
 If needed, you can run the container with a different command:
 
 ```sh
-docker run -it --rm -p 3000:3000 -v /path/to/course:/course prairielearn/prairielearn COMMAND
+docker run -it --rm -p 3000:3000 -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn COMMAND
 ```
 
 This can be used to, e.g., run scripts distributed with PrairieLearn.
+
+#### Development from the shell
+
+When making local changes to server-side code, it is faster to restart only the node server instead of the whole docker container.  This can be done by starting the container into a shell environment and starting the server manually:
+
+```sh
+docker run -it --rm -p 3000:3000 -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn /bin/bash
```

Comment:
Maybe change this to just initialize the bash session instead of running the script?

From there, go into a brief overview of the possible `docker` scripts described by @mwest1066 ?

```bash
docker run -it --rm -p 3000:3000 -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn /bin/bash
# inside container:
cd /PrairieLearn
npm ci
./docker/start_postgres.sh
./docker/lint_js.sh
./docker/lint_python.sh
./docker/test_js.sh
./docker/test_python.sh
```